catalog/ui/search-ui/standard/src/main/webapp/js/view/ingest/IngestModal.view.js

```diff
@@ -37,8 +37,6 @@ define([
             },
             initialize: function () {
                 // there is no automatic chaining of initialize.
-                Modal.prototype.initialize.apply(this, arguments);
```

Comment:
We can't do this here. Removing this line causes the view to not be cleaned up. But we create a new modal in the IngestMenu.js file, which then causes this view to become a zombie. I think we need to look at the `isUnfinished` function and figure out why that isn't keeping the original modal around.